admin_guide/managing_projects.adoc

```diff
@@ -309,3 +309,26 @@ in order, this rule should be specified last.
 ====
 
 endif::openshift-origin,openshift-enterprise,atomic-registry[]
+
+[[recovering-deleted-default-projects]]
+
+== Recovering Deleted Default Projects
+
+If you delete any of the default projects like `*default*`, `*openshift*`, or
+`*openshift-infra*`, then you will likely experience issues that impact the
+whole environment. To recover from this, restart the master service:
+
+====
+----
+# systemctl restart atomic-openshift-master
```

Comment:
after restarting, verify the projects exist and are not in the terminating state:
```
oc get namespace/default namespace/openshift namespace/openshift-infra
```

If they are still terminating, wait for the termination to complete, then restart the master service again.